app/serializers/concerns/image_serializable.rb

```diff
@@ -5,11 +5,11 @@ module ImageSerializable
 
   included do
     attribute :image, if: Proc.new { |the_object| the_object.image.attached? } do |the_object|
-      helpers.rails_blob_url(the_object.image)
+      eval "helpers.#{the_object.class.to_s.underscore}_image_large_url(the_object)"
```

Comment:
I'm pretty sure `send` is preferable to eval for safety and performance reasons.
```suggestion
      helpers.send("#{the_object.class.to_s.underscore}_image_large_url", the_object)
```